app/models/concerns/fake_auth_config.rb

```diff
@@ -0,0 +1,27 @@
+module FakeAuthConfig
+  # Used in an initializer to determine if the application is configured and allowed to use fake authentication.
+  def self.fake_auth_status
+    return true if fake_auth_enabled? && app_name_pattern_match?
+
+    false
+  end
+
+  # Default to fake auth in development unless FAKE_AUTH_ENABLED=false. This allows rake tasks to run without loading
+  # ENV.
+  private_class_method def self.fake_auth_enabled?
```

Comment:
I actually changed it to `private`, but rubocop got mad at me. Apparently, `private` doesn't work for class methods. (I think it should, but who am I to say!)

Reply:
Oh right... makes sense now. I was missing that these were all class methods and just focused on "why is this weird" when it turns out it is not weird, we just normally don't make class methods private because, well, maybe that's the weird bit going on here lol. It's totally fine as we know it works and is never actually run in prod anyway so :shrug: